data/json/monsters/nether_animal.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "id": "mon_cat_mutant_prism",
+    "type": "MONSTER",
+    "copy-from": "mon_cat",
+    "name": { "str": "iridescent cat" },
+    "description": "As it moves, this cat's colors shift unnaturally in the light.  You can't make out the breed, but its fur pattern seems to change with every step.",
+    "//": "Higher dodge due to the constantly changing fur pattern",
+    "dodge": 9
+  },
+  {
+    "id": "mon_cat_liquid",
+    "type": "MONSTER",
+    "copy-from": "mon_cat",
+    "name": { "str": "liquid cat" },
+    "description": "It is universally known that cats are liquid, but this cat's body is actually quivering while it walks around.  It also enjoys resting while twisting its body in ways that would break a normal cat's bones.",
+	"phase": "LIQUID",
```

Comment:
cant really make a multiline suggestion but this and the next line needs to be indented.

Reply:
Whoops! I forgot to lint! Thanks for catching this!